### PR TITLE
e2e tests for dictionary management

### PR DESCRIPTION
**Pull request by mugurrus**
SD-1825 Implement e2e tests for dictionary management

### CODE REVIEW

File: client/spec/dictionaries_spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+var openUrl = require('./helpers/utils').open,
+    dictionaries = require('./helpers/dictionaries');
+
+describe('DICTIONARIES', function() {
+    'use strict';
+
+    describe('edit dictionary', function() {
+        beforeEach(function() {
```
Comment (petrjasek) [inline client/spec/dictionaries_spec.js]: you can move this one level up and it will run for both edit and delete blocks

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

File: client/spec/dictionaries_spec.js (hunk 1 above)
Comment (mugurrus) [inline client/spec/dictionaries_spec.js]: I know, but it makes sense to group tests based on major features.

**Pull request merged**